app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+    module V1
+      class NotesController < Api::V1::ApiController
+
+        def create
+          @note = Project.find(params[:project_id]).notes.create!(note_params)       
+        end
+  
+        def update
+          @note = Note.includes(:project).find(params[:id])
+          @note.update!(note_params)
+        end
+
+        def destroy
+            note = Note.find(params[:id]).destroy!
+            head :ok
+        end
+
+        private
+
+        def note_params
+            params.require(:note).permit(:text)
+        end
+
+      end
+    end
+  end
+  
```

Comment:
This controller should be in a folder named project, the endpoint should be /api/vi/project/:id/notes
https://guides.rubyonrails.org/routing.html